src/App.tsx

```diff
@@ -1,203 +1,106 @@
+/* eslint-disable import/extensions */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './Components/TodoList';
+import { Footer } from './Components/Footer';
```

Comment:
```suggestion
import { Footer } from './components/Footer';
```
fix everywhere

Do not ignore such errors - try to figure it out or ask for help in the chat
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/86923089/31cb6405-db1e-49ca-9cd0-6a99e95218c4)